base.js

```diff
@@ -33,8 +33,6 @@ module.exports = {
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     // Enforce import order
     'import/order': 'error',
-    // Imports should come first
-    'import/first': 'error',
```

Comment:
that's import/order vs import/first. 2 different rules.

Reply:
It is duplicate with standard:

https://github.com/standard/eslint-config-standard/blob/f7d50e01310866458fd844625cf588d4ee39823b/eslintrc.json#L190

Reply:
Still prefer being explicit.